Pressure those in power page

AaronBowles opened the pull request:
Good afternoon, using the change-how-you-eat as a template I have created a pressure those in power page. I added the page and its link to the choices grid but for now it just uses the EatLogo. I added a links to two different websites where you can find and contact political officials. The visited logic was also implemented.

Commit:
for pull

rachel-harris1 commented on src/pages/pressure-those-in-power/index.js:
```diff
@@ -0,0 +1,315 @@
+  <Layout>
+    <SEO
+      title="Pressure those in power to help prevent climate change"
+      description="Climate change is an urgent crisis that cannot be held seperate from politics for real change to occur"
```
small typo: `separate`

rachel-harris1 commented on src/pages/pressure-those-in-power/index.js:
```diff
@@ -0,0 +1,315 @@
+    <SEO
+      title="Pressure those in power to help prevent climate change"
+      description="Climate change is an urgent crisis that cannot be held seperate from politics for real change to occur"
+      image="https://climatechoice.co/images/Seo/eat.jpg"
```
would be fab to get an image that is specific to this page cc @shylands

philsturgeon replied:
agreed, if you want to grab something off https://stock.adobe.com I'll grab it. I have a bajillion credits.

philsturgeon replied:
Pick anything from https://stock.adobe.com and I'll grab it

rachel-harris1 commented on src/pages/pressure-those-in-power/index.js:
```diff
@@ -0,0 +1,315 @@
+        `global warming solutions`,
+        `Climate change the facts`,
+        `climate change solutions`
+      ]}
```
fab 👍 👍

rachel-harris1 commented on src/pages/pressure-those-in-power/index.js:
```diff
@@ -0,0 +1,315 @@
+          >
+            <li>
+              Those with political power make decisions for the world that
+              effect emissions and the state of our planet.
```
I believe this should read `affect emissions`

rachel-harris1 commented on src/pages/pressure-those-in-power/index.js:
```diff
@@ -0,0 +1,315 @@
+              </li>
+              <li>
+                Many people vote based on political affiliation regardless of
+                policies the politicain holds.
```
small typo: `politician`

rachel-harris1 commented on src/pages/pressure-those-in-power/index.js:
```diff
@@ -0,0 +1,315 @@
+                policies the politicain holds.
+              </li>
+              <li>
+                Often people do not know where to start or believe that they
```
💯  can relate 💯

rachel-harris1 commented on src/pages/pressure-those-in-power/index.js:
```diff
@@ -0,0 +1,315 @@
+              the politicans whos policies most directly affect your day to day
+              life. This also lays the groundwork for changing the planet from
+              the bottom-up.
+            </p>
```
This reads really well 👍

rachel-harris1 commented on src/pages/pressure-those-in-power/index.js:
```diff
@@ -0,0 +1,315 @@
+              How much would this help?
+            </h3>
+            <p className={`${ChoiceStyles.paragraph} ${Styles.paragraph}`}>
+              If we are able to elect officails who value our planet and hope to
```
`officials`

rachel-harris1 commented on src/pages/pressure-those-in-power/index.js:
```diff
@@ -0,0 +1,315 @@
+              If we are able to elect officails who value our planet and hope to
+              minimize or eliminate the effects humans have on the climate, then
+              their policies would be reflective of that. This would incentivise
+              industry to change with their ways or get left behind. No true
```
`the industry`

rachelfranklin1 reviewed (commented):
This is really great work, thank you for taking the time to write this up and build the page!

It would be great if we could have a few localised versions of usa.gov/elected-officials for other nations - even a handful eg for the UK, USA, and any other countries that you could find. 

Do you think this is possible?

AaronBowles commented:
Hey Rachel,

I think understand what you're saying but just to double check. I already have a localized version for contacting officials (local/any and congress reps) for the US but you would like me to add some additional countries? If so that will definitely be possible!

Commit:
added german and italian contacts

Commit:
added canadian, indian and australian contacts

Commit:
fixed minor typo

AaronBowles commented:
Good afternoon, I have added localized contacts for UK, France, Spain, Germany, Italy, Canada, India and Australia. I hope this is a good start and I can add more in the future. Let me know if you need anything else and I hope this page can go live!

AaronBowles commented:
Also, do you need me to close this pull request and create another or would you be able to merge all the newly added additions from here?

philsturgeon commented:
There's some content/links on here you could use protect.earth/politics

AaronBowles commented:
Thanks Phil, what exactly would you like me to add? I could add a link to protect.earth/politics?

philsturgeon commented:
@AaronBowles its got links for contact senators and MPs which could be helpful, but yeah protect earth is aiming to be a bit of an international directory of action all over the world, whilst this site is more of a guide of concepts. So if you'd like to link to protect earth for "Find protests, petitions and important votes near you!" then that'd be great.

Commit:
added president of US

Commit:
added president contact and adjusted wording for any elected official contact

Commit:
added links to ProtectEarth, NASA, OCeanic Preservation Society and Stand For Trees. Added link to browse various resources with world.org and enviromentalsciencedegree.com

AaronBowles commented:
@philsturgeon I added a link to contact the president to accompany the senators link already there. I don't know if you saw but I also have contacts for elected officials in UK, France, Spain, Germany, Italy, Canada, India and Australia. I have just added the link to Protect.earth. Additionally, I added links to visit NASA, the Oceanic Preservation Society, Stand For Trees, and two sites with lists of climate change sites and web resources to learn more on climate change. I know those last few are not specific to pressure those in power but I thought it might be helpful resources for those looking to get more involved. Let me know what you think

philsturgeon commented on src/pages/pressure-those-in-power/index.js:
```diff
@@ -0,0 +1,544 @@
+              affect emissions and the state of our planet.
+            </li>
+            <li>
+              Lobbiest for big oil and other large energy companies use their
```
```suggestion
              Lobbists for big oil and other large energy companies use their
```

AaronBowles replied:
I believe after a quick search it should be lobbyists

philsturgeon replied:
there was a typo in my suggestion hahaha yes

Commit:
corrected spelling Lobbiests to Lobbyist

Commit:
corrected spelling Lobbyist to Lobbyists

philsturgeon commented:
This is getting really close to done, great work. 

Little bits include 
<img width="388" alt="Screen Shot 2020-03-11 at 6 13 11 PM" src="https://user-images.githubusercontent.com/67381/76444340-1b3ddb80-63c4-11ea-8304-55e64ae45f0f.png">

philsturgeon commented on src/pages/pressure-those-in-power/index.js:
```diff
@@ -0,0 +1,544 @@
+              <b>Reach out to public officials!</b> All congressmen and
+              congresswomen have public contacts. <b>Start local!</b> Before you
+              try and fix a nation, fix your own backyard. Local officials are
+              the politicans whos policies most directly affect your day to day
```
```suggestion
              the politicians who's policies most directly affect your day to day
```

philsturgeon commented on src/pages/pressure-those-in-power/index.js:
```diff
@@ -0,0 +1,544 @@
+            </h2>
+            <p className={`${ChoiceStyles.paragraph} ${Styles.paragraph}`}>
+              The biggest impact can be had if you{" "}
+              <b>investigate the polcies of those up for election</b>, then vote
```
```suggestion
              <b>investigate the policies of those up for election</b>, then vote
```

philsturgeon commented on src/pages/pressure-those-in-power/index.js:
```diff
@@ -0,0 +1,544 @@
+              Lobbyists for big oil and other large energy companies use their
+              immense budget to influence politicians, causing politicians who
+              do take climate change seriously and seek a solution severely
+              underfunded comparitively.
```
```suggestion
              underfunded comparatively.
```

philsturgeon commented on src/pages/pressure-those-in-power/index.js:
```diff
@@ -0,0 +1,544 @@
+            <p className={`${ChoiceStyles.paragraph} ${Styles.paragraph}`}>
+              If we are able to elect officials who value our planet and hope to
+              minimize or eliminate the effects humans have on the climate, then
+              their policies would be reflective of that. This would incentivise
```
```suggestion
              their policies would be reflective of that. This would incentivize
```

Commit:
corrected spelling errors

AaronBowles commented:
@philsturgeon I have corrected the spelling changes, do you need me to create the new banner image? I was thinking of a gavel for the page icon then using it for the banner image.

AaronBowles commented:
@philsturgeon https://stock.adobe.com/images/judge-gavel-or-auction-hammer-icon/143940067?prev_url=detail 

This could be a good banner image, let me know what you think

philsturgeon commented:
@AaronBowles ok I went with a different image but we're nearly there. I've got a local copy of this branch and im sneaking a few bits in. 

@shylands how did you make these SVG's src/configs/pages/choiceEatConfig.js

Also what was the data source for the 26% on the food svg? This resource is great https://informationisbeautiful.net/visualizations/how-to-reduce-the-worlds-carbon-footprint-by-2050/ 

We need to try and figure out what to show for guides that dont have a specific % number, like "getting politicians to change policies" depends on how many policies and well well they're enacted and enforced, would be hard to have a simple pie chart for it. Maybe we can have another type of image, or just the happy earth icon without a pie slice.

shylands commented:
@philsturgeon I just designed them all pulled together from different free vector components. 

No idea what the data source was for the eat one, would have just been some article I came across at the time while researching :)

AaronBowles commented:
@philsturgeon Sounds good, sneak in whatever is needed!

philsturgeon commented on src/pages/pressure-those-in-power/index.js:
```diff
@@ -0,0 +1,544 @@
+import ButtonStyles from "../../styles/Buttons.module.scss";
+import Styles from "./Styles.module.scss";
+import ShareSocialCta from "../../components/ShareSocialCta";
+import shareConfig from "../../configs/pages/choiceEatConfig";
```
This share information is still from the Eat guide.

philsturgeon commented on src/pages/pressure-those-in-power/index.js:
```diff
@@ -0,0 +1,544 @@
+import ShareSocialCta from "../../components/ShareSocialCta";
+import shareConfig from "../../configs/pages/choiceEatConfig";
+import ChoiceGridOnChoice from "../../components/ChoiceGrid/onChoice";
+import BannerImg from "../../components/Images/Graphics/choiceBannerEat.svg";
```
@shylands I know you're a busy man but could you knock out a banner image with a gavel and mark it at 40%? 

> The EU is also close to completing a comprehensive legislative framework that will enable it to implement its commitment under the Paris Agreement to cut emissions by at least 40% by 2030.
https://ec.europa.eu/clima/news/eu-conclude-ratification-second-kyoto-protocol-commitment-period-end-year_en

If not could you help me find somebody who can? I have no idea what to do with a SVG.

richsoni replied:
I can update this image, but also wondering if it might be useful to make these banner svg's a component instead.

We could use something like this https://www.npmjs.com/package/react-minimal-pie-chart to overlay the pie chart on top of the world image.  That would make this stuff configurable without rolling a new SVG from scratch each time

philsturgeon replied:
@richsoni that would be amazing. Think you could make another PR which either extends this one, or we do that first and then update this one? Either way, this one has been sat around for ages and we should get it merged. It's great work.

richsoni replied:
https://github.com/impactmakers/climatechoice/pull/155/files

richsoni replied:
That pr updates the SVG, and the config file.  I can put together a separate PR after that to do the dynamic pie charts

philsturgeon replied:
@richsoni thank you! I'll close this one down and move the two remaining bits of feedback over there.

philsturgeon commented on src/pages/pressure-those-in-power/index.js:
```diff
@@ -0,0 +1,544 @@
+    <SEO
+      title="Pressure those in power to help prevent climate change"
+      description="Climate change is an urgent crisis that cannot be held separate from politics for real change to occur"
+      image="https://climatechoice.co/images/Seo/eat.jpg"
```
Ignore this I've got a good image committed locally.

philsturgeon commented on src/components/ChoiceGrid/items.js:
```diff
@@ -196,6 +198,31 @@ const ChoiceGridItems = () => {
             </div>
           </Link>
         </li>
+        <li className={`${Styles.choiceGridItem} ${Styles.choiceEat}`}>
```
This may also be incorrect? Lots of eat mentions.

AaronBowles replied:
I need the choicePower style and the powerIcon to adjust the logic from the Eat logic in the choicegrid component

philsturgeon commented:
This is getting handled in https://github.com/impactmakers/climatechoice/pull/155, thank you so much @AaronBowles !
